documentation/src/main/resources/api.yaml

```diff
@@ -33,21 +33,21 @@ paths:
       parameters:
         - name: name
           in: query
-          required: true
+          required: false
           description: Name. If name is "NOT_VALID" an validation error occurs.
           schema:
             type: string
             maxLength: 10
           example: 'John Doe'
-      summary: Call the thugs service
-      description: Post your name and get some hugs or meet some thugs.
+      summary: Get name.
+      description: Post a name and get all instances of the name in the db. Post without a name to get everything that's in the db.
```

Comment:
Jeg synes ikke vi skal kalde det post i beskrivelsen. For mig indikerer det at det er en HTTP post der laves og det er det ikke i det her tilfælde.